_posts/2019-06-05-hibernate-search-extension.adoc

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: 'New extension: Hibernate Search + Elasticsearch'
+date: 2019-06-05
+tags: quarkus hibernate elasticsearch
+author: gsmet
+---
+
+In the 0.15.0 version of Quarkus, we introduced the Hibernate Search + Elasticsearch extension and, just a few days ago, we added link:/guides/hibernate-search-guide[a guide] and https://github.com/quarkusio/quarkus-quickstarts/tree/master/hibernate-search-elasticsearch[a quickstart] explaining how to use it.
+
+Note that this extension is still experimental as it is based on an Alpha version of Hibernate Search 6.
+While the code is stable and thoroughly tested, some features are still missing and we might change some APIs along the way to the final version - even if the Hibernate team has made a great effort to stabilize them before the inclusion in Quarkus.
+
+If you don't mind updating your code from time to time, you're very welcome to become an early adopter and provide feedback.
+
+== What is Hibernate Search?
+
+This is an old time favorite of mine.
+Back in the days, when I was developing business applications for customers, discovering Hibernate Search really changed the way we implemented search engines and our customers were very happy with the search features we provided from that moment on.
+I soon started to contribute to it and... here I am.
+
+With an additional dependency and some configuration, you end up being able to provide full text search features to your users, just by synchronizing your Hibernate ORM entities to a full text index.
+
+And when I say full text search, I mean the real one, backed by http://lucene.apache.org/[Apache Lucene] or https://www.elastic.co/[Elasticsearch].
+
+== How do I use it?
+
+It can't be easier.
+
+First, add the `quarkus-hibernate-search-elasticsearch` extension to your Quarkus application:
+
+[source, shell]
+----
+./mvnw quarkus:add-extension -Dextension=hibernate-search-elasticsearch
+----
+
+(or you can directly add the `io.quarkus:quarkus-hibernate-search-elasticsearch` dependency to your `pom.xml`)
```

Comment:
you can leave it but remember this is hyper opinionated so people can learn the alternatives in guides

Reply:
Yeah, I left it because using the Maven command reformats your pom file.